apps/condo/domains/meter/hooks/useMeterTableColumns.tsx

```diff
@@ -51,6 +59,63 @@ const METER_READING_INPUT_ADDON_STYLE: CSSProperties = {
     color: colors.sberGrey[6],
 }
 
+const MeterReadingDatePicker = ({ record, newMeterReadings, setNewMeterReadings }: MeterReadingDatePickerType) => {
+    const intl = useIntl()
+    const MissedVerificationTooltip = intl.formatMessage({ id: 'pages.condo.meter.MissedVerification.tip' })
+
+    const handleInputContainerClick = useCallback(e => e.stopPropagation(), [])
+    const meterId = get(record, ['meter', 'id'])
+    const lastReadingDate = get(record, 'lastMeterReadingDate', dayjs().toISOString())
+    const nextVerificationDate = get(record, ['meter', 'nextVerificationDate'], '')
+    const pickerValue = get(newMeterReadings, [meterId, 'date'], dayjs().toISOString())
```

Comment:
Since you have added typing, could you also get rid of the `get` function where possible? so that errors are displayed at the application build stage and good types are output

For typescript, it is better to use optional chains instead of `get`

Reply:
good point, fixed it where possible